### PR TITLE
forme -> form

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: asm/scrcmd_move_tutor.s [hunk 1]
```diff
@@ -196,9 +196,9 @@ _02202C74:
 	.balign 4, 0
 	thumb_func_end ScrCmd_656
 
-	thumb_func_start SpeciesAndFormeToWazaOshieIndex
-SpeciesAndFormeToWazaOshieIndex: ; 0x02202C7C
-	; Convert species and forme into
+	thumb_func_start SpeciesAndFormToMoveTutorIndex
+SpeciesAndFormToMoveTutorIndex: ; 0x02202C7C
+	; Convert species and formeinto
```
Comment (adrienntindall) [inline asm/scrcmd_move_tutor.s:201]: messed up the comments here but it's nbd

File: files/fielddata/wazaoshie/waza_oshie.json.txt [hunk 2]
```diff
@@ -40,14 +40,14 @@
...
+gMoveTutorMoves:
 ## for set in tutor
     oshie_begin
 ## for move in set.moves
     oshie_add {{ move }}
 ## endfor
     oshie_end
 ## endfor
-    .size gWazaOshieMoves,.-gWazaOshieMoves
+    .size gMoveTutorMoves,.-gMoveTutorMoves
```
Comment (adrienntindall) [inline files/fielddata/wazaoshie/waza_oshie.json.txt:53]: can we change this file's name too while we're at it?

File: src/pokemon_icon_idx.c [hunk 3]
```diff
@@ -2,20 +2,20 @@
...
-    int forme;
+    int form;
     u32 ret;
 
     encry = AcquireBoxMonLock(boxMon);
     species = GetBoxMonData(boxMon, MON_DATA_SPECIES, NULL);
     isEgg = GetBoxMonData(boxMon, MON_DATA_IS_EGG, NULL);
-    forme = _BoxMonGetForme(boxMon);
-    ret = GetMonIconNaixEx(species, isEgg, forme);
+    form = _BoxMonGetForm(boxMon);
```
Comment (adrienntindall) [inline src/pokemon_icon_idx.c:17]: Any reason why there's a missing prefix here?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

File: src/pokemon_icon_idx.c (hunk 3 above)
Comment (red031000) [reply src/pokemon_icon_idx.c:17]: genuinely no idea

File: files/fielddata/wazaoshie/waza_oshie.json.txt (hunk 2 above)
Comment (red031000) [reply files/fielddata/wazaoshie/waza_oshie.json.txt:53]: that might be a bit more difficult, since the stuff is passed into o2narc iirc

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**